magenta/models/music_vae/js/demo/index.ts

```diff
@@ -1,40 +1,77 @@
 // tslint:disable-next-line:max-line-length
-import { MusicVAE, Note, DrumsConverter, DrumRollConverter, MelodyConverter } from '../index';
+import { MusicVAE, Note, DrumsConverter, DrumRollConverter, MelodyConverter, INoteSequence} from '../index';
```

Comment:
I think I still may not be fully understanding the NoteSequence vs INoteSequence stuff. Why use INoteSequence here and NoteSequence within the other files?